samples/tapi/hog.cpp

```diff
@@ -76,10 +76,8 @@ int main(int argc, char** argv)
         "{ s scale     | 1.0            | resize the image before detect}"
         "{ o output    |                | specify output path when input is images}";
     CommandLineParser cmd(argc, argv, keys);
-    if (cmd.has("help"))
```

Comment:
That would change the intended functionality I think. There are defaults for all the values so that the application can run out of the box. The problem is that if whether one uses the -h switch or not, the help text ALWAYS prints. In fact, the application doesn't do anything except print the help text right now, for any combination of inputs.

The intention of this line is to get the value for help. cmd.has("help") does not achieve this. My change does.